src/CommonITILValidation.php

```diff
@@ -197,9 +197,30 @@ public static function canValidate($items_id)
         $iterator = $DB->request([
             'SELECT' => ['users_id_validate'],
             'FROM'   => static::getTable(),
+            'LEFT JOIN' => [
+                ValidatorSubstitute::getTable() => [
+                    'FKEY' => [
+                        ValidatorSubstitute::getTable() => 'users_id',
+                        static::getTable() => 'users_id_validate',
+                    ],
+                ],
+                User::getTable() => [
+                    'FKEY' => [
+                        ValidatorSubstitute::getTable() => 'users_id',
+                        User::getTable() => 'id',
+                    ],
+                ],
+            ],
             'WHERE'  => [
                 static::$items_id    => $items_id,
-                'users_id_validate'  => Session::getLoginUserID()
+                'OR' => [
+                    'users_id_validate'  => Session::getLoginUserID(),
+                    'AND' => [
+                        User::getTable() . '.substitution_start_date' => ['<=', $_SESSION['glpi_currenttime']],
+                        User::getTable() . '.substitution_end_date' => ['>=', $_SESSION['glpi_currenttime']],
+                        ValidatorSubstitute::getTable() . '.users_id_substitute' => Session::getLoginUserID(),
+                    ],
+                ]
```

Comment:
add the substitutes of the validator to the users allowed to validate an ITIL object. Takes into account the date range set by the delegator.